CITATION.cff

```diff
@@ -33,6 +33,6 @@ keywords:
   - deep-learning
   - annotation
 license: MIT
-version: 2.2.0
-date-released: '2022-09-12'
+version: 2.8.2
```

Comment:
Most probably we want to update our update_version.py script to do that automatically.